src/backend/marsha/core/models/video.py

```diff
@@ -382,3 +410,62 @@ def get_source_s3_key(self, stamp=None):
         return "{video!s}/thumbnail/{pk!s}/{stamp:s}".format(
             video=self.video.pk, pk=self.pk, stamp=stamp
         )
+
+
+class LiveRegistration(BaseModel):
+    """Model representing a scheduling live registration."""
+
+    RESOURCE_NAME = "liveregistrations"
+
+    email = models.EmailField(_("email address"))
+
+    lti_consumer_site = models.ForeignKey(
```

Comment:
I think we are calling it just "consumer_site" elsewhere in the code base:
```suggestion
    consumer_site = models.ForeignKey(
```

Reply:
it was to specify that it's only in lti case

Reply:
IC. But consumer_site is only for LTI anyway?

Reply:
yes! I've changed it everywhere.